### PR TITLE
Decrease importance of Core category

### PR DESCRIPTION
**Pull request by emmanuelbernard**
Also force YAML and JSON as pinned.

### CODE REVIEW

**Comment by emmanuelbernard** [comment]
Ping @maxandersen @aloubyansky @ia3andy

File: bom/runtime/src/main/resources/extensions-overrides.json [hunk 1]
```diff
@@ -47,6 +42,16 @@
         ]
       }
     },
+    {
+      "name": "Core",
+      "id": "core",
+      "description": "Essential Quarkus components. Provided automatically",
```
Comment (ia3andy) [inline bom/runtime/src/main/resources/extensions-overrides.json]: @emmanuelbernard is it still/always true that they are provided automatically (ex `logging-json` and `logging-sentry`)?

Comment (emmanuelbernard) [reply bom/runtime/src/main/resources/extensions-overrides.json]: Ah good point, something like 

"Basic Quarkus components: engine, log, etc." is probably a better description. Can you push it @ia3andy ?

**Comment by gsmet** [comment]
CI was failing and the build wasn't available anymore. I force pushed an update.

@ia3andy could you check @emmanuelbernard 's comment and also be sure Quarkus.io is OK with it?

File: bom/runtime/src/main/resources/extensions-overrides.json (hunk 1 above)
Comment (ia3andy) [inline bom/runtime/src/main/resources/extensions-overrides.json]:
```suggestion
      "description": "Basic Quarkus components: engine, log, etc.",
```

Comment (ia3andy) [reply bom/runtime/src/main/resources/extensions-overrides.json]: @emmanuelbernard you just have to accept the change..

**Comment by ia3andy** [comment]
> CI was failing and the build wasn't available anymore. I force pushed an update.
> 
> @ia3andy could you check @emmanuelbernard 's comment and also be sure code.quarkus.io is OK with it?

@gsmet I've suggested the description. It should be ok with code quarkus, I can test it tomorrow if you think that's necessary?

**Comment by gsmet** [comment]
The JSON file is invalid anyway.

**Comment by gsmet** [comment]
I fixed the JSON file and adjusted the description. What bugs me a bit is that we are referencing very old Camel Quarkus versions in this file. Is it expected? /cc @aloubyansky

**Comment by ia3andy** [comment]
@maxandersen might know for Camel Quarkus

**Comment by gsmet** [comment]
I tested code.quarkus.io, it seems to be OK so I will merge as soon as CI is green.

**Comment by maxandersen** [comment]
the platform is referencing the version of camel the camel team stated quarkus platform should point to.

**Comment by maxandersen** [comment]
@lburgazzoli should the platform be updated to something more recent in prep for quarkus 1.2 release ?

**Comment by lburgazzoli** [comment]
@maxandersen I think we can remove the camel bits from this file as we have the metadata on camel side now

**Comment by maxandersen** [comment]
merged as all green

**Comment by ia3andy** [comment]
@maxandersen is this part of the 1.2.0.Final?

**Comment by ia3andy** [comment]
I am asking because on 1.2.0.Final, the descriptor still give me categories with core first.

**Comment by maxandersen** [comment]
no, it was for some reason not merged in in time.

@gsmet should/could we do a 1.2.1 platform release ?

**Pull request closed without merging**